cypress/support/helper-functions/participant/participant-helpers.js

```diff
@@ -0,0 +1,22 @@
+export const timeout = Cypress.env('timeout');
+
+export function signInWithClever() {
+  cy.wait(0.2 * timeout);
+  cy.get('button', { timeout: timeout }).contains('Clever').click();
+
+  cy.origin('https://clever.com/oauth/authorize', () => {
+    cy.get('input[title="School name"]', { timeout: 60000 })
+      .type('61e8aee84cf0e71b14295d45')
```

Comment:
I might make all of these hard-coded strings (i.e. '61e8aee84cf0e71b14295d45') into .env variables in the Cypress config file, in case we ever want to change them or re-use them.

Reply:
Good point, I've added them as env variables.